### ISSUE TITLE
Can't convert TIFF

### ISSUE THREAD

**Issue by dcorrigan** [h2non/node-imaginary#6]
Hi,

I deployed imaginary to a small heroku instance here: https://dan-test5.herokuapp.com

I can use it to convert JPEG, PNG, etc., fine, but have trouble with TIFF images. I'm just using the sample scripts from the README, so my code is just:

```
var fs = require('fs')
var imaginary = require('imaginary')
var serverUrl = 'https://dan-test5.herokuapp.com'

imaginary(fs.createReadStream('./sample.tiff'))
  .server(serverUrl)
  .rotate({ rotate: 180 })
  .on('error', function (err) {
    console.error('Cannot resize the image:', err)
  })
  .pipe(fs.createWriteStream('out.jpg'))
```

This produces a 415 from the server. Am I missing something obvious? Does the server need to be compiled with extra binaries for TIFF support? Thanks for any info.

**Comment by h2non** [comment]
Yes, TIFF support may require additional server support, but perhaps the image type cannot be properly inferred.

Would you mind to share the image?

**Comment by dcorrigan** [comment]
Sure. I get a 415 with any TIFF I try, but an example is attached in a zip file. Thanks very much for looking!

[tiff-sample.zip](https://github.com/h2non/node-imaginary/files/396922/tiff-sample.zip)

**Comment by dcorrigan** [comment]
Hi, just checking in. Did you have a chance to look at that sample? Thanks!

**Comment by h2non** [comment]
TIFF type is not allowed a valid type in `imaginary`. This should be probably fixed.

**Comment by dcorrigan** [comment]
It looks like this is fixed on the 0.1.27 release of `imaginary`. Thanks very much @h2non !

**Comment by h2non** [comment]
Yes! Enjoy it, and please report any issue you may experiment.
